src/gui/ignorelisttablewidget.h

```diff
@@ -18,12 +18,12 @@ class IgnoreListTableWidget : public QWidget
     IgnoreListTableWidget(QWidget *parent = nullptr);
     ~IgnoreListTableWidget() override;
 
-    void readIgnoreFile(const QString &file, bool readOnly = false);
-    int addPattern(const QString &pattern, bool deletable, bool readOnly);
+    void readIgnoreFile(const QString &file, const bool readOnly = false);
+    int addPattern(const QString &pattern, const bool deletable, const bool readOnly);
```

Comment:
please remove the `const` qualifier for POD data types as they must not be in the declaration interface for the methods, only the implementation

they do not change the signature for a caller (the generated symbol in the shared library does not include them)
they are an implementation detail

Reply:
Thanks, fixed